### DISCUSSION TITLE
Favicon doesn't work

### DISCUSSION THREAD

**Discussion by Vagabonding** [bluecaret/carettab#386] [Help and Support]
Hi, is there anyway to fix the issue that the favicons never show? They will show very briefly when you have the website open but then disappear again. Maybe there could be an option to save a logo/favicon so it doesn't have to rely on getting it from the website? Thanks.

**Comment by bluecaret** [comment]
Helllo, I'm sorry you're having trouble with the favicons, the way I have to gather those icons is very finicky. For starters the icons are tied to your temporary files. So if you have your browser set to clear those out automatically or you are clearing them out manually, this will cause CaretTab to lose access to them and they won't appear until you've visited the site again. In addition to this they just been not very reliable. The way I access favicons for a site is via the official guidelines that Google provides, unfortunately they didn't seem to do a good job of making it.

For now all I can help you with is to check to make sure your temporary files are not being automatically deleted.

At some point down the road I may add the option to allow users to add their own icon for a link. I don't know when I'll be able to get to adding that feature though.

Sorry this isn't a better option. Thank you for using CaretTab!

**Comment by Vagabonding** [comment]
Hi, Thanks for the reply. 

I use Brave and have it set to clear all browsing history when I close it, but even when I don't close it the favicons seem to disappear very soon after the site is closed, I'm not sure why. It would be great if there was an option to save a logo in a separate folder and simply have the tab pick up the logo from there instead of using the site favicon from temp folder. The favicons are generally also small and poor resolution, so they don't look good on a bigger screen even when they do work. But I understand, of course, if this is not a priority for you to fix. I still love CareTab and will keep using it. :) Thanks again. 


On 14-Dec-24 21:48:54, John Hancock ***@***.***> wrote:
Helllo, I'm sorry you're having trouble with the favicons, the way I have to gather those icons is very finicky. For starters the icons are tied to your temporary files. So if you have your browser set to clear those out automatically or you are clearing them out manually, this will cause CaretTab to lose access to them and they won't appear until you've visited the site again. In addition to this they just been not very reliable. The way I access favicons for a site is via the official guidelines that Google provides, unfortunately they didn't seem to do a good job of making it.
For now all I can help you with is to check to make sure your temporary files are not being automatically deleted.
At some point down the road I may add the option to allow users to add their own icon for a link. I don't know when I'll be able to get to adding that feature though.
Sorry this isn't a better option. Thank you for using CaretTab!
